### PR TITLE
Fix the positioning issue of the Facebook icon and the Apply button.

### LINKED ISSUE #65: In Home page "Apply" button is not visible
**Describe the bug**
In Home page "Apply" button is not visible 

**To Reproduce**
1. Go to Home page
2. Click on "Apply" button
3. When user is hover that button then it will visible 

**Expected behavior**
That "Apply" button need to visible clearly
**Screenshots**
Screenshot_20240104_123941

**Desktop (please complete the following information):**
 - OS: Fedora Linux 39 (KDE Plasma) x86_64 
 - Browser Brave 
 - Version v.1.61.109

**Smartphone (please complete the following information):**

**Additional context**

**Are you working on it?**

### PR DESCRIPTION
**Pull request by mayura-andrew**
## Purpose

The purpose of this PR is to fix #65 

## Goals

The goal is to improve the user interface by fixing the positioning of the Facebook icon and the Apply button.

## Approach

The approach taken was to modify the CSS styles of the Facebook icon and the Apply button.

## Screenshots

Screenshot_20240109_234644-1

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
 
N/A

## Test environment
 
OS - Fedora Linux 39 (KDE Plasma) x86_64
NodeJS - 20.10.0

## Learning

N/A

### CODE REVIEW

File: src/components/Home/Home.tsx [hunk 1]
```diff
@@ -26,8 +26,10 @@ const Home: React.FC = () => (
         <div style={{ marginTop: '40px' }}>
           <Button
             className={styles.homePageMainButton}
-            type={'primary'}
-            style={{ marginRight: '20px', marginBottom: '20px' }}
+            style={{
+              marginRight: '20px',
+              marginBottom: '20px',
+            }}
```
Comment (anjula-sack) [inline src/components/Home/Home.tsx]: Let's use tailwind css here @mayura-andrew

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/components/Home/Home.tsx (hunk 1 above)
Comment (mayura-andrew) [reply src/components/Home/Home.tsx]: okay sure

Comment (mayura-andrew) [reply src/components/Home/Home.tsx]: https://github.com/sef-global/scholarx-frontend/assets/48531182/1d9bdfb1-a8f4-4a23-a51c-6435417cf8aa

File: src/components/Home/Home.tsx [hunk 2]
```diff
@@ -24,14 +24,18 @@ const Home: React.FC = () => (
...
-          </Button>
-          <Button className={styles.homePageMainButton}>See more</Button>
+          <div className="flex mt-10">
+            <Button
+              className={`flex justify-center items-center mr-5 mb-5 bg-blue-500 hover:bg-blue-700 text-white py-1 px-16 rounded w-40 h-9 font-sans font-bold`}
+            >
+              Apply
+            </Button>
+            <Button
+              className={`flex justify-center items-center mr-5 mb-5 text-black py-1 px-16 rounded w-40 h-9 font-sans font-bold`}
```
Comment (anjula-sack) [inline src/components/Home/Home.tsx]: We don't need template literals here.

```suggestion
              className="flex justify-center items-center mr-5 mb-5 text-black py-1 px-16 rounded w-40 h-9 font-sans font-bold"
```

File: src/components/Home/Home.tsx [hunk 3]
```diff
@@ -24,14 +24,18 @@ const Home: React.FC = () => (
...
-            className={styles.homePageMainButton}
-            type={'primary'}
-            style={{ marginRight: '20px', marginBottom: '20px' }}
-          >
-            Apply
-          </Button>
-          <Button className={styles.homePageMainButton}>See more</Button>
+          <div className="flex mt-10">
+            <Button
+              className={`flex justify-center items-center mr-5 mb-5 bg-blue-500 hover:bg-blue-700 text-white py-1 px-16 rounded w-40 h-9 font-sans font-bold`}
```
Comment (anjula-sack) [inline src/components/Home/Home.tsx]:
```suggestion
              className="flex justify-center items-center mr-5 mb-5 bg-blue-500 hover:bg-blue-700 text-white py-1 px-16 rounded w-40 h-9 font-sans font-bold"
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/components/Home/Home.tsx (hunk 2 above)
Comment (mayura-andrew) [reply src/components/Home/Home.tsx]: Ok got it 👍

**Review by anjula-sack — APPROVED** [review]
Good job! @mayura-andrew

**Comment by mayura-andrew** [comment]
Thank you :)

**Pull request merged**